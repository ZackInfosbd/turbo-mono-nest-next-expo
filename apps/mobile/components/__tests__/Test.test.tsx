import * as React from 'react';
import renderer from 'react-test-renderer';
import Test from '../ui/test';
import { Text } from 'react-native';

it(`renders correctly`, () => {
  const tree = renderer
    .create(
      <Test>
        <Text>Snapshot test for children!</Text>
      </Test>,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
