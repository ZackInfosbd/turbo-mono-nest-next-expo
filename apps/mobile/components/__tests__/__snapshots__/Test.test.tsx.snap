// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    className=" text-red-200 font-bold text-xl p-2"
  >
    there is something needs to be here. Or not!
  </Text>
  <Text>
    Snapshot test for children!
  </Text>
</View>
`;
